app/build.gradle

```diff
@@ -264,16 +263,21 @@ android {
     kotlinOptions {
         jvmTarget = '1.8'
     }
-
-    // Room - exclude the atomic functions module from the package and prevent warnings.
     packagingOptions {
-        exclude 'META-INF/atomicfu.kotlin_module'
+        pickFirst("META-INF/atomicfu.kotlin_module")
```

Comment:
I assume exclude and pickFirst has different behaviors. Do you mind elaborating on this change, why it's necessary, and what it improves?

Reply:
The project had no support for building on Apple Silicon, Kotlin plugin needed to be updated and with that this error appeared:
https://stackoverflow.com/questions/55702672/build-failed-after-update-coroutines-to-1-2-0-meta-inf-atomicfu-kotlin-module